src/tmxrasterizer/tmxrasterizer.cpp

```diff
@@ -116,6 +118,8 @@ bool TmxRasterizer::shouldDrawLayer(const Layer *layer) const
 int TmxRasterizer::render(const QString &fileName,
                           const QString &imageFileName)
 {
+    PluginManager::instance()->loadPlugins();
```

Comment:
I'd prefer if this was put in the `main` function rather than in the `TmxRasterizer`.

Reply:
Will do. I'll drop it in right after the app initialization calls in main unless you have another convention of where it should go.